<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Sql</class>
 <widget class="QWidget" name="Sql">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>685</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 淡棕色温暖背景 - QWidget主窗口 */
QWidget {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1,
                                stop:0 #f5e6d3, stop:0.5 #f0d9c0, stop:1 #ebccad);
    border: none;
}

/* 分组框样式 - 白色卡片 */
QGroupBox {
    background-color: rgba(255, 255, 255, 0.9);
    border: 1px solid #d4b996;
    border-radius: 8px;
    margin-top: 10px;
    padding-top: 10px;
    color: #5c3713;
    font-weight: bold;
}

QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 0 8px;
    background-color: #ffffff;
    color: #8b5a2b;
    border-radius: 4px;
}

/* Frame样式 */
QFrame {
    background-color: rgba(255, 255, 255, 0.85);
    border: 1px solid #d4b996;
    border-radius: 6px;
}

/* 普通标签样式 */
QLabel {
    background-color: transparent;
    color: #5c3713;
    font-size: 14px;
}

/* 特殊标签 - 时间显示等 */
QLabel#time_1 {
    color: #8b4513;
    font-size: 16px;
    font-weight: bold;
}

/* 状态标签 */
QLabel#labelDUANKAI,
QLabel#label_6wIFI {
    color: #d2691e;
    font-weight: bold;
}

/* 输入框样式 - 淡粉色 */
QLineEdit {
    background-color: #FFE4E6;  /* 淡粉色 */
    border: 1px solid #f8c6cb;
    border-radius: 4px;
    padding: 6px 8px;
    color: #5c3713;  /* 棕色文字 */
    font-size: 14px;
    selection-background-color: #f8c6cb;
}

QLineEdit:focus {
    border: 1px solid #f4a7b0;
    background-color: #ffdde0;
}

/* 纯文本编辑框 - 网易云黑 */
QPlainTextEdit {
    background-color: #2d2d2d;  /* 网易云黑 */
    border: 1px solid #404040;
    border-radius: 4px;
    padding: 6px 8px;
    color: #e0e0e0;  /* 浅灰色文字 */
    font-size: 14px;
    selection-background-color: #1db954;
}

/* 表格视图 - 网易云黑 */
QTableView {
    background-color: #2d2d2d;  /* 网易云黑 */
    border: 1px solid #404040;
    border-radius: 4px;
    color: #e0e0e0;
    gridline-color: #404040;
    alternate-background-color: #333333;
}

QTableView::item {
    background-color: #2d2d2d;
    color: #e0e0e0;
    border: none;
    padding: 4px;
}

QTableView::item:selected {
    background-color: #1db954;
    color: #ffffff;
}

QTableView::item:hover {
    background-color: #3a3a3a;
}

QHeaderView::section {
    background-color: #404040;
    color: #e0e0e0;
    padding: 6px;
    border: none;
    font-weight: bold;
}

/* 复选框样式 */
QCheckBox {
    background-color: transparent;
    color: #5c3713;
    spacing: 8px;
}

QCheckBox::indicator {
    width: 16px;
    height: 16px;
    border: 1px solid #b08f6d;
    border-radius: 3px;
    background-color: #ffffff;
}

QCheckBox::indicator:checked {
    background-color: #d2691e;
    border: 1px solid #d2691e;
}

/* 滑动条样式 */
QSlider::groove:horizontal {
    border: 1px solid #b08f6d;
    height: 4px;
    background-color: #e8d5c4;
    border-radius: 2px;
}

QSlider::handle:horizontal {
    background-color: #d2691e;
    border: 1px solid #d2691e;
    width: 16px;
    height: 16px;
    border-radius: 8px;
    margin: -6px 0;
}

QSlider::handle:horizontal:hover {
    background-color: #e37c2b;
    border: 1px solid #e37c2b;
}

/* 按钮样式 - 保持原有粉色效果 */
QPushButton,
QToolButton {
    border: none;
    background-color: transparent;
    outline: none;
    padding: 8px 12px;
    border-radius: 4px;
    font-size: 14px;
    color: #5c3713;  /* 棕色文字 */
}

QPushButton:hover,
QToolButton:hover {
    background-color: #FFE4E6;  /* 淡粉色 */
    border: none;
    color: #333333;
}

QPushButton:pressed,
QToolButton:pressed {
    background-color: #F3E8FF;  /* 淡紫色 */
    border: none;
    color: #333333;
}

QPushButton:focus,
QToolButton:focus {
    outline: none;
    border: none;
}

/* 特殊按钮样式 */
QPushButton#clear_yu_bt,
QPushButton#set_yu_bt,
QPushButton#set_light_bt {
    background-color: #d4b996;
    color: #5c3713;
}

QPushButton#clear_yu_bt:hover,
QPushButton#set_yu_bt:hover,
QPushButton#set_light_bt:hover {
    background-color: #e0c9a8;
}

/* 线条分隔符 */
Line {
    background-color: #d4b996;
    border: none;
}

/* 进度条样式 */
QProgressBar {
    border: 1px solid #b08f6d;
    border-radius: 4px;
    text-align: center;
    color: #5c3713;
    background-color: #e8d5c4;
}

QProgressBar::chunk {
    background-color: qlineargradient(x1:0, y1:0, x2:1, y2:0,
                                      stop:0 #d2691e, stop:1 #e37c2b);
    border-radius: 3px;
}</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>52</y>
     <width>691</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>2</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
</string>
   </property>
   <property name="lineWidth">
    <number>10</number>
   </property>
   <property name="text">
    <string>数据库管理平台</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="start_l">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>511</y>
     <width>41</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>开始</string>
   </property>
  </widget>
  <widget class="QPushButton" name="now_bt">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>539</y>
     <width>50</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(132, 66, 198);
border-radius:10px;		 
background-color: rgb(170, 85, 255);
color: rgb(255, 255, 255);
font-size:15px; 		 


</string>
   </property>
   <property name="text">
    <string>NOW</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>520</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(100, 100, 194);
border-radius:10px;		 
background-color: rgb(129, 129, 194);
color: rgb(255, 255, 255);



</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>502</y>
     <width>3</width>
     <height>86</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_bt">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>484</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(59, 59, 208);   
font: 700 11pt &quot;微软雅黑&quot;;
border-radius:10px;		 
background-color: rgb(85, 85, 255);
color: rgb(255, 255, 255);			

</string>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>539</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="date">
    <date>
     <year>2022</year>
     <month>10</month>
     <day>7</day>
    </date>
   </property>
   <property name="time">
    <time>
     <hour>20</hour>
     <minute>50</minute>
     <second>0</second>
    </time>
   </property>
   <property name="displayFormat">
    <string>yyyy-MM-dd HH:mm:ss</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>490</y>
     <width>211</width>
     <height>181</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(80, 229, 229);
border-radius:10px;		 
background-color: rgb(170, 255, 255);
color: rgb(0, 0, 0);
font-size:15px; 		 

</string>
   </property>
   <property name="title">
    <string>值查询</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>580</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(140, 66, 0);
border-radius:10px;		 
background-color: rgb(198, 66, 0);
color: rgb(255, 255, 255);
 


</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QPushButton" name="select_val_bt">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>630</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>AcadEref</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(0, 133, 120);
border-radius:10px;		 
background-color: rgb(0, 170, 255);	
color: rgb(255, 255, 255);
font-size:15px; 		 

</string>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>580</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(140, 66, 0);
border-radius:10px;		 
background-color: rgb(198, 66, 0);
color: rgb(255, 255, 255);
 


</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>479</y>
     <width>341</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border:2px solid rgb(151, 151, 227);
border-radius:10px;		 
background-color: rgb(170, 170, 255);
color: rgb(0, 0, 0);
font-size:16px; 		 

</string>
   </property>
   <property name="title">
    <string>日期查询</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <widget class="QPushButton" name="currDay_bt">
    <property name="geometry">
     <rect>
      <x>2</x>
      <y>130</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid rgb(0, 120, 180);
border-radius:10px;		 
background-color: rgb(0, 170, 127);
color: rgb(255, 255, 255);
font-size:15px; 		 


</string>
    </property>
    <property name="text">
     <string>最近一天</string>
    </property>
   </widget>
   <widget class="QPushButton" name="currThDay_bt">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid rgb(63, 63, 190);
border-radius:10px;		 
background-color: rgb(85, 85, 255);
color: rgb(255, 255, 255);
font-size:15px; 		 


</string>
    </property>
    <property name="text">
     <string>最近三天</string>
    </property>
   </widget>
   <widget class="QPushButton" name="currWeek_bt">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>130</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid rgb(180, 0, 180);
border-radius:10px;		 
background-color: rgb(255, 0, 255);
color: rgb(255, 255, 255);
font-size:15px; 		 




</string>
    </property>
    <property name="text">
     <string>最近一周</string>
    </property>
   </widget>
   <widget class="QLabel" name="start_l_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>35</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QLabel" name="end_l">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>75</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>楷体</family>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>结束</string>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEditStart_2">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>27</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateTimeEdit" name="dateTimeEditEnd_2">
    <property name="geometry">
     <rect>
      <x>43</x>
      <y>70</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="select_data_bt">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>20</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid rgb(0, 133, 120);
border-radius:10px;		 
background-color: rgb(0, 170, 255);	
color: rgb(255, 255, 255);
font-size:15px; 		 


</string>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="now_bt_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>70</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border:2px solid rgb(132, 66, 198);
border-radius:10px;		 
background-color: rgb(170, 85, 255);
color: rgb(255, 255, 255);
font-size:15px; 		 


</string>
    </property>
    <property name="text">
     <string>NOW</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="icon">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>12</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/img/data_nie.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="displayAll_bt">
   <property name="geometry">
    <rect>
     <x>359</x>
     <y>532</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(180, 150, 0);   
border-radius:10px;		 
background-color: rgb(255, 170, 0);
color: rgb(255, 255, 255);			
font-size:15px; 		 
</string>
   </property>
   <property name="text">
    <string>显示所有</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>501</y>
     <width>3</width>
     <height>86</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="clear_bt">
   <property name="geometry">
    <rect>
     <x>359</x>
     <y>580</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>-1</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(134, 134, 0);   
border-radius:10px;		 
background-color: rgb(170, 170, 0);
color: rgb(255, 255, 255);			
</string>
   </property>
   <property name="text">
    <string>清空数据</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonExls">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>630</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 2px solid rgb(134, 134, 0);   
background-color: rgb(255, 0, 127);
border-radius:10px;		 
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>导出</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPicture">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 0, 127);
font: 700 12pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>显示历史曲线图</string>
   </property>
  </widget>
  <zorder>tableView</zorder>
  <zorder>title</zorder>
  <zorder>start_l</zorder>
  <zorder>now_bt</zorder>
  <zorder>line</zorder>
  <zorder>exit_bt</zorder>
  <zorder>dateTimeEdit_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>spinBox_2</zorder>
  <zorder>select_val_bt</zorder>
  <zorder>spinBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>icon</zorder>
  <zorder>displayAll_bt</zorder>
  <zorder>line_2</zorder>
  <zorder>clear_bt</zorder>
  <zorder>comboBox</zorder>
  <zorder>pushButtonExls</zorder>
  <zorder>pushButtonPicture</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
